get-shit-done/bin/lib/installer-migrations.cjs

```
@@ -0,0 +1,665 @@
+    return { ...result, plan };
+  } catch (error) {
+    primaryError = error;
+    throw error;
+  } finally {
+    try {
+      releaseLock();
+    } catch (releaseError) {
+      if (primaryError) {
+        primaryError.suppressed = [...(primaryError.suppressed || []), releaseError];
+      } else if (completed) {
+        throw releaseError;
```

Biome (2.4.14)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)